### PR TITLE
make small range input smaller

### LINKED ISSUE #851: Make small range input smaller
We've been running into cases where we need a smaller range input. I think we should be find just shrinking the existing s range input a bit rather than adding a new size.

### PR DESCRIPTION
**Pull request by samanpwbb**
New smaller range input:

screen shot 2017-09-11 at 12 10 05 pm

closes #851

### CODE REVIEW

**Comment by samanpwbb** [comment]
Note: when building the repo, I noticed some mysteriously uncommitted changes to color-variants. These should have been committed, so I committed them here.

File: src/forms.css [hunk 1]
```diff
@@ -517,24 +517,33 @@ because it seems to be more cross-browser consistent with all HTML5 input types
...
+  width: 12px;
+  height: 12px;
+  margin-top: -5px;
 }
 
 .range--s > input::-ms-thumb {
-  width: 16px;
-  height: 16px;
+  width: 8px;
+  height: 8px;
```
Comment (dasulit) [inline src/forms.css]: These are smaller in edge, is that on purpose?

Comment (samanpwbb) [reply src/forms.css]: Accident! Just fixed.

**Review by dasulit — APPROVED** [review]
Great! I like this new size, and ranges look good across browsers 👍  I don't know how to evaluate the color-variant diff, though, so... :woman_shrugging:

**Pull request merged**